HEN_HOUSE/egs++/sources/egs_radionuclide_source/egs_radionuclide_source.h

```diff
@@ -255,9 +234,8 @@ results for non-disintegration emissions.
 :stop geometry definition:
 :start source definition:
     :start source:
-        name                = my_source
-        library             = egs_radionuclide_source
-        activity            = 28e6
+        name                = source_location
+        library             = egs_isotropic_source
         geometry            = my_envelope
         region selection    = IncludeSelected
         selected regions    = 1 2
```

Comment:
Although I now see the base source (source_location), I don't see the egs_radiohead_source (i.e. my_source) defined in the updated example.  Is this missing?

Reply:
You're right! I've fixed this.

Reply:
Thanks, @rtownson!